django_q/admin.py

```diff
@@ -43,7 +43,7 @@ def retry_failed(FailAdmin, request, queryset):
 class FailAdmin(admin.ModelAdmin):
     """model admin for failed tasks."""
 
-    list_display = ("name", "func", "started", "stopped", "short_result")
+    list_display = ("name", "func", "started", "stopped", "short_result", "attempt_count")
```

Comment:
I would remove the attempt count column. I'm not sure it adds anything extra cause it will always show the configured max amount of tries. Maybe at some point we can add attempt_count as a kwarg on each individual task to override the global settings. Then it would make more sense to me.